src/main/java/com/google/jenkins/plugins/credentials/oauth/RemotableGoogleCredentials.java

```diff
@@ -70,7 +79,7 @@ public RemotableGoogleCredentials(
       throw new GeneralSecurityException(
           Messages.RemotableGoogleCredentials_NoAccessToken(), e);
     }
-    this.accessToken = checkNotNull(credential.getAccessToken());
+    this.accessToken = mock ? "MOCKED" : checkNotNull(credential.getAccessToken());
     this.expiration = new DateTime().plusSeconds(
```

Comment:
I would rather just see this field converted to a simple `long` (msec since epoch), in which case `JodaDateTimeConverter` would be purely for compatibility with old settings and its `marshal` could throw `UnsupportedOperationException`.